config/model_configs/target_amip_albedo_function.yml

```diff
@@ -0,0 +1,23 @@
+run_name: "target_amip_n1_shortrun"
```

Comment:
this file is largely copied from `config/longrun_configs/amip_n1_shortrun.yml`, but using functional bucket albedo and with `t_end` decreased from 1 day to 300 seconds (2*dt)